builtin/audit/file/backend_test.go

```diff
@@ -0,0 +1,79 @@
+package file
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/vault/audit"
+	"github.com/hashicorp/vault/helper/salt"
+)
+
+func TestAuditFile_fileModeNew(t *testing.T) {
+	salter, _ := salt.NewSalt(nil, nil)
+
+	modeStr := "0777"
+	mode, err := strconv.ParseUint(modeStr, 8, 32)
+
+	path, err := ioutil.TempDir("", "test")
```

Comment:
It'd be nicer to any user trying to clean up anything that is still left (for instance, in panic cases deferred functions won't be run) if the prefix wasn't `"test"` -- something like `"vault-test_audit_file-file_mode_new"` would be much more descriptive.